services/metadata/src/metadata/services/avus.clj

```diff
@@ -34,6 +35,7 @@
       (when (not-empty new-avus)
         (persistence/add-metadata-template-avus user-id new-avus data-type))
       (persistence/set-template-instances data-id template-id (map :id avus)))
+    (amqp/publish-metadata-update user-id data-id)
```

Comment:
These publishings have to happen after the transaction has committed. I don't know of any functionality which would let me put this inside the transaction and have it sent when the transaction commits, which is what would be preferable. But, if this ends up sending extra messages, that's not too much of a concern either; the indexing is fast and always updates it to the current state, whatever that state is.